### DISCUSSION TITLE
I don't believe the 2nd law of thermodynamics. (The most uplifting video I'll ever make.)

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#62] [Astrophysics and Physics]
To counter claims that the universe will [end in a disaster](https://www.astrokatie.com/book), Sabine Hossenfelder explains how for one system considered by scientists ("the universe") entropy does not change.

On YouTube: [I don't believe the 2nd law of thermodynamics. (The most uplifting video I'll ever make.)](https://youtu.be/89Mq6gmPo0s)
It's a click bait title, what she means is "*I don't believe the 2nd law of thermodynamics can be applied to the universe.*"

Her argument is that thermodynamics works well for atoms and steam engines, but for extremely large systems the components (microstates) evolve more quickly than the time they need to interact when the system (macrostates) becomes excessively large.

For example, in a steam engine (for which thermodynamics was developed) the speed of sound carries information across the engine in a few milliseconds, a time comparable to the changes of configuration experienced by the engine itself.

However in cosmology, galaxies evolve significantly over tens of Gyrs, but even the fastest interaction (light propagating between galaxies) takes much more time to cover those large distances between galaxies.  When light leaves a galaxy, it will likely take hundreds of Gyrs before it is absorbed in another galaxy.  Thermodynamical entropy loses its meaning in such huge systems.

We can forget the idea of a "heat death" of the universe.

**Comment by ftessier** [comment]
Good point! There are two additional problems with applying thermodynamics to the whole universe:

1. entropy is perspectival: it depends on the "detector" (the ability to resolve microstates); and
2. thermodynamics is not covariant, it is (normally) derived in the rest frame of the _gedanken_ box of gas. According to the [Maxwell-Jüttner](https://en.wikipedia.org/wiki/Maxwell%E2%80%93J%C3%BCttner_distribution) derivation, temperature is invariant. What about entropy?

**Reply by RedshiftDrift** [reply]
According to [this StackExchange-Physics answer](https://physics.stackexchange.com/a/275252/362675), they cite a paper that claims that
>  a "general consensus about this matter has not been reached"

I haven't read the paper.

**Reply by ftessier** [reply]
Thanks for the reference. Einstein and Plank pondered on the question of relativistic thermodynamics as early 1907, it seems. But the question remains controversial after 100+ years. I stumbled on this long-standing blind spot in physics while trying to determine temperature in a molecular dynamics simulation a long while ago! I was surprised that no professor ever noted the tacit "rest-frame" assumption in classical thermodynamics (I think I eventually found _one_ text book that mentions it).

Here is a recent _Nature_ paper with a clear title as can be, which seems to cite most of the references I had collected on this subject:

https://www.nature.com/articles/s41598-017-17526-4#ref-CR3

_(I have not read this paper either 😉)_

Although I am no thermodynamics expert — and barely know anything about cosmology — it always struck me that this oversight might imply important consequences for our conception of the observable universe. Don't we apply thermodynamic principles, for example, to determine the temperature of distant stars, that are or might be in motion relative to our frame of reference?

**Reply by ftessier** [reply]
I remember that I found this PRL article particularly illuminating, as the authors resorted to a "real" molecular dynamics simulation of ideal relativistic particles:

https://journals.aps.org/prl/pdf/10.1103/PhysRevLett.99.170601

> The numerical evidence unequivocally favors the Jüttner distribution. Moreover, our simulations illustrate that the concept of “thermal equilibrium” extends naturally to special relativity only if a many-particle system is spatially confined. They make evident that “temperature” can be statistically defined and measured in an observer frame independent way.

_(This one I read!)_

**Reply by ftessier** [reply]
Woah, this PRL has garnered a slew of citations from recent articles on this subject, time for me to get back into it! I note the following title: [Analysis of the multicomponent relativistic Boltzmann equation for electron scattering in big bang nucleosynthesis](https://journals.aps.org/prd/abstract/10.1103/PhysRevD.101.123532). 

**Spoiler:** 

> ... the predictions of standard BBN remain unchanged. 😞

**Reply by RedshiftDrift** [reply]
> Don't we apply thermodynamic principles, for example, to determine the temperature of distant stars, that are or might be in motion relative to our frame of reference?

I don't think it's a problem in that case because the different parts (gas, plasma) of the system (star) do not have relativistic speeds with respect to each other.  A Galilean transformation to our frame of reference keeps the conclusion valid.

However,
> ***relativistic*** Boltzmann equation for electron scattering in big bang nucleosynthesis

is necessary because during the primordial nucleosynthesis of a hypothetical Big Bang, the temperatures/energies are certainly in the relativistic regime.

**Comment by budrap00** [comment]
That's correct. Given the scale of the Cosmos we observe and the finite
maximum light speed, the concept of the "whole universe" has no physical
meaning. It's not that we lack the ability to observe it - it doesn't
exist.

The concept of the whole universe as a unitary system is a foundational
assumption of the standard model. The FLRW metric is a "universal"
framework. Slapping it on GR which doesn't have a universal frame was an
oxymoronic exercise. Garbage in, garbage out as the old aphorism has it.The
expanding universe model is the garbage out.

Entropy is a 19th century laboratory concept that arose in the study of
closed systems. The Cosmos is not a closed system. It is not a system at
all.

There are no closed systems in the Cosmos. Entropy is only half true for
all finite systems like a galaxy which has to first wind up (syntropy)
before it can unwind (entropy). All thermodynamical and biological systems
work that way.
